learn/index.md

```diff
@@ -1,8 +1,93 @@
 ---
```

Comment:
I didn't review this page because I know we talked about modifying it slack so will wait for those changes to be made before giving this page a look 👀